Automattic/wc-calypso-bridge
# Free trial extension: customize payment tasklist header

Partially fixes https://github.com/Automattic/wc-calypso-bridge/issues/945

## Changes in this PR

### WooCommerce Payments - tasklist

1. Header title - Change the copy from “It’s time to get paid” to “It’s time to test payments”
2. Header description - Change the copy from “[...] Verify your business details to start managing transactions with WooCommerce Payments” to “[...] Verify your business details to start testing transactions with WooCommerce Payments.”
3. Cta - Change copy from “Verify details” to “Test payments”.

### Payments task - tasklist

1. Header title - Change the copy from “It’s time to get paid” to “It’s time to test payments”
2. Header description - Change copy to "Give your customers an easy and convenient way to pay! Set up one (or more!) of our fast and secure online or in person payment methods."
3. Cta - Change copy from “Verify details” to “Test payments”.

## Test instructions

1. Make sure to copy over the latest changes in [woocmmerce-calypso-bridge-helper.php](https://gist.github.com/moon0326/cac46c70a2cee81b61faef517fef7178) and the plugin is activated.
2. Change [this line](https://gist.github.com/moon0326/cac46c70a2cee81b61faef517fef7178#file-a-woocommerce-calypso-bridge-helper-php-L33) to false in your local env. This forces your env to have free trial.
3. Go to WooCommerce > Home
4. Complete store details, add product task
5. Observe that the payment task list header matches the new copy
6. Install WCpay and choose a WCpay eligible  country
7. Go to WooCommerce > Home
8. Observe that the WCPay task list header matches the new copy

Payments task

<img src="https://user-images.githubusercontent.com/4344253/219277573-9b55aaba-f6c7-4177-b26c-814fc81e5f11.png" width="600px" />

WCPay task

<img src="https://user-images.githubusercontent.com/4344253/219277582-76f29475-2e0c-48d3-8f28-019d0de5cb4f.png" width="600px" />
